### PR TITLE
Add peertube support

### PR DESCRIPTION
**Pull request by Murmele**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -27,5 +27,17 @@ which is the video ID:
...
+
+   .. peertube:: 327a21b3-374e-4373-8b2c-494c9f5e1f19
+
+Custom Server for peertube instances:
+
+.. code-block:: rst
+
+   .. peertube:: 327a21b3-374e-4373-8b2c-494c9f5e1f19
+
+      :server: peertube.tv
```
Comment (Murmele) [inline README.rst]:
```suggestion
      :instance: peertube.tv
```

File: docs/usage.rst [hunk 2]
```diff
@@ -105,6 +113,11 @@ To start the video at a specific time the parameter "url_parameters" may be used
    .. vimeo:: 486557682
       :url_parameters: "#t=0m43s"
 
+.. code-block:: rst
+
+   .. peertube:: 486557682
+      :server_url: "peertube.tv"
```
Comment (Murmele) [inline docs/usage.rst]:
```suggestion
      :instance: "peertube.tv"
```

File: tests/roots/test-video/peertube.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+peertube
+========
+
+.. peertube:: 327a21b3-374e-4373-8b2c-494c9f5e1f19
+
+.. peertube:: 327a21b3-374e-4373-8b2c-494c9f5e1f19
+   :server_url: "peertube.tv"
```
Comment (Murmele) [inline tests/roots/test-video/peertube.rst]:
```suggestion
   :instance: "peertube.tv"
```

**Comment by Murmele** [comment]
@dham

File: docs/usage.rst [hunk 4]
```diff
@@ -105,6 +125,11 @@ To start the video at a specific time the parameter "url_parameters" may be used
    .. vimeo:: 486557682
       :url_parameters: "#t=0m43s"
 
+.. code-block:: rst
+
+   .. peertube:: 486557682
```
Comment (Murmele) [inline docs/usage.rst]:
```suggestion
   .. peertube:: 327a21b3-374e-4373-8b2c-494c9f5e1f19
```

File: docs/usage.rst [hunk 5]
```diff
@@ -105,6 +125,11 @@ To start the video at a specific time the parameter "url_parameters" may be used
...
+      :instance: "peertube.tv"
```
Comment (Murmele) [inline docs/usage.rst]:
```suggestion
      :url_parameters: "?start=0s"
```

File: docs/usage.rst [hunk 6]
```diff
@@ -120,6 +145,10 @@ In LaTeX output, the following code will be emitted for the videos:
 
    \sphinxcontribvimeo{https://player.vimeo.com/video/}{148751763}{"#t=0m43s"}
 
+.. code-block:: latex
+   
+   \sphinxcontribPeerTube{https://peertube.tv/w/}{327a21b3-374e-4373-8b2c-494c9f5e1f19}{?start=43}
```
Comment (dham) [inline docs/usage.rst]: To be consistent, please use lower case in the macro name.

**Review by dham — CHANGES_REQUESTED** [review]

File: docs/usage.rst (hunk 6 above)
Comment (dham) [inline docs/usage.rst]:
```suggestion
   \sphinxcontribpeertube{https://peertube.tv/w/}{327a21b3-374e-4373-8b2c-494c9f5e1f19}{?start=43}
```

File: tests/test_build.py [hunk 7]
```diff
@@ -42,11 +53,16 @@ def test_latex(app, status, warning):
 
     assert r"\newcommand{\sphinxcontribyoutube}" in result
     assert r"\newcommand{\sphinxcontribvimeo}" in result
+    assert r"\newcommand{\sphinxcontribPeerTube}" in result
```
Comment (dham) [inline tests/test_build.py]:
```suggestion
    assert r"\newcommand{\sphinxcontribpeertube}" in result
```

File: tests/test_build.py [hunk 8]
```diff
@@ -42,11 +53,16 @@ def test_latex(app, status, warning):
...
 
     assert r"\sphinxcontribyoutube{https://youtu.be/}{dQw4w9WgXcQ}{}" in result
     assert (
         r"\sphinxcontribvimeo{https://player.vimeo.com/video/}{148751763}{}" in result
     )
+    assert (
+        r"\sphinxcontribPeerTube{https://peertube.tv/w/}{327a21b3-374e-4373-8b2c-494c9f5e1f19}{}"
```
Comment (dham) [inline tests/test_build.py]:
```suggestion
        r"\sphinxcontribpeertube{https://peertube.tv/w/}{327a21b3-374e-4373-8b2c-494c9f5e1f19}{}"
```

**Review by dham — CHANGES_REQUESTED** [review]

File: sphinxcontrib/youtube/peertube.py [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+    pass
+
+
+class PeerTube(utils.Video):
+    """Custom version of the Video Directive."""
+
+    _node = peertube
+    _thumbnail_url = "{}.jpg"
+    _platform = "PeerTube"
```
Comment (dham) [inline sphinxcontrib/youtube/peertube.py]:
```suggestion
    _platform = "peertube"
```

**Review by dham — APPROVED** [review]

**Comment by Murmele** [comment]
Sorry I was on holiday. Shall I change anything?

**Comment by dham** [comment]
As you can see from the commit record, the changes were minor so I made them myself using GithHub's suggestion feature. This has now been merged and is in the 1.4 release on PyPI. Thanks for your contribution!

**Comment by Murmele** [comment]
Thanks for merging!

**Pull request merged**